readme.md

```diff
@@ -16,16 +16,13 @@ Information about the Intel® RealSense™ technology at [www.intelrealsense.com
 
 > :open_file_folder: Don't have access to a RealSense camera? Check-out [sample data](./doc/sample-data.md)
 
-## Special notice from Intel® RealSense™ regarding the recent press announcement
+## Update on Recent Changes to the RealSense Product Line
```

Comment:
Those changes are not expected if you are based on hkr branch.
those are from development branch.
What happened?